### ISSUE TITLE
Wrong pixel data written to file in full-frame mode

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/xpedaq#128]
The basic phenomenology is that the pixels at indices 8 and 9 (44, 0 and 44, 1) in the matrix have unphysical pulse height values in the output files. A short test with the pedestal application seems to indicate that the values are perfecty reasonable in the TDS, so it must be somthing happening when we actually write stuff to file.

**Comment by lucabaldini** [comment]
This might be connected with issue #127.

**Comment by albertomanfreda** [comment]
It seems that the reason of this bug is that we are overwriting the content of two pixels when writing the timestamp in the events, which should not be done in full-frame mode. Should be easy to fix (but likely not connected with issue #127)

**Comment by albertomanfreda** [comment]
Fixed with the change:

```
*/
 void pDataBlock::setStartSeconds(unsigned int startSeconds)
 {
+  if (!m_isWindowed) return;
   if (errorSummary() == 0) {
     for (unsigned int evt = 0; evt < numEvents(); evt ++) {
       unsigned int offset = m_offsetVec[evt] + Seconds;
```

which means that right now we do not have a timestamp for events in full-frame mode, which in the future will go into the file header (see issue  #86).
